### PR TITLE
Docker - etcd - BUMP to 3.5.15 and added debian12

### PR DESCRIPTION
**Pull request by AdrianManiek**

### CODE REVIEW

File: templates/Dockerfile.template [hunk 1]
```diff
@@ -1,85 +1,68 @@
...
+ENV GOPATH="/go"
+ENV PATH="$GOPATH/bin:$PATH"
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin"
+
+# Install govendor
+RUN go install github.com/kardianos/govendor@latest
+
+# Configure environment variables for etcd
+ARG ETCD_VERSION=3.5.15
```
Comment (armandomiani) [inline templates/Dockerfile.template:23]: Version is hard-coded in template, use the versions.yaml package variables

File: templates/Dockerfile.template [hunk 2]
```diff
@@ -1,85 +1,68 @@
...
-
-RUN set -eux \
-    # Extracts licences
-    && cd "${GOPATH}/src/github.com/etcd-io/etcd" \
-    && govendor license +vendor > /NOTICES
+RUN apt-get update && \
+    apt-get install -y wget git tar gzip build-essential
+
+# Install Go
+ARG GO_VERSION=1.20
```
Comment (armandomiani) [inline templates/Dockerfile.template:8]: add golang to the packages and use the package variables

**Review by armandomiani — CHANGES_REQUESTED** [review]

File: templates/Dockerfile.template (hunk 1 above)
Comment (AdrianManiek) [reply templates/Dockerfile.template]: Fixed. Package variables have been used.

File: templates/Dockerfile.template (hunk 2 above)
Comment (AdrianManiek) [reply templates/Dockerfile.template]: Fixed. Golang was added to the versions.yaml and have been used into the image.

**Review by armandomiani — APPROVED** [review]

**Pull request merged**